monoio/src/runtime.rs

```diff
@@ -351,9 +351,6 @@ impl From<Runtime<TimeDriver<IoUringDriver>>> for FusionRuntime<TimeDriver<IoUri
 ///
 /// # Examples
 ///
-/// In this example, a server is started and `spawn` is used to start a new task
-/// that processes each received connection.
-///
```

Comment:
[This is a copy-paste error](https://docs.rs/tokio/latest/tokio/task/fn.spawn.html#examples), since the provided example is not a TCP error, let's just remove this sentence.